### PR TITLE
add support for domainHint

### PR DESCRIPTION
**Pull request by jindrichkuba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ const {MsAuthPlugin} = Plugins;
 const result = await MsAuthPlugin.login({
     clientId: '<client id>',
     tenant: '<tenant, defaults to common>',
+    domainHint: '<domainHint, defaults to common>',
```
Comment (bartwesselink) [inline README.md:80]: Is it correct that the domain hint defaults to common? Or does it not default to anything?

File: README.md [hunk 2]
```diff
@@ -93,6 +94,7 @@ const {MsAuthPlugin} = Plugins;
 await MsAuthPlugin.logout({
     clientId: '<client id>',
     tenant: '<tenant, defaults to common>',
+    domainHint: '<domainHint, defaults to common>',
```
Comment (bartwesselink) [inline README.md:97]: Same here.

**Review by bartwesselink — COMMENTED** [review]
Nice work! Thank you for your contribution. I have one minor question though.

File: README.md (hunk 1 above)
Comment (jindrichkuba) [reply README.md:80]: yes, you are right, I think default should be anything

File: README.md (hunk 2 above)
Comment (jindrichkuba) [reply README.md:97]: yes

**Comment by bartwesselink** [comment]
Added in release 3.1.1, note that it is ignored on iOS as it is not supported in the current MSAL version.

**Pull request merged**